profiler/src/ProfilerEngine/Datadog.Profiler.Native/ContentionProvider.cpp

```diff
@@ -81,7 +81,7 @@ void ContentionProvider::OnContention(double contentionDurationNs)
     {
         std::lock_guard lock(_contentionsLock);
 
-        if (!_sampler.Sample(bucket))
+        if (!_sampler.Sample(bucket, contentionDurationNs))
```

Comment:
can you `static_cast` to a `std::uint64_t` to remove the warning please :)

Reply:
✅ was already done  ;^)